staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -723,3 +735,18 @@ func containsString(needle string, haystack []string) bool {
 	}
 	return false
 }
+
+// exemptNamespaceWarning returns a non-empty warning message if the exempt namespace has a
+// non-privileged policy and sets pod security labels.
+func (a *Admission) exemptNamespaceWarning(exemptNamespace string, policy api.Policy) string {
+	if policy.Enforce.Level == api.LevelPrivileged &&
+		policy.Warn.Level == api.LevelPrivileged &&
+		policy.Audit.Level == api.LevelPrivileged {
+		return ""
+	} else if policy.Equivalent(&a.defaultPolicy) {
+		return ""
+	}
+
+	return fmt.Sprintf("namespace %s is exempt from Pod Security, and the policy %s will be ignored",
```

Comment:
nit: quote namespace to match other invalid request errors:

```suggestion
	return fmt.Sprintf("namespace %q is exempt from Pod Security, and the policy %s will be ignored",
```

Reply:
Done. Also quoted in a few a other places.

As an aside, I was thinking about changing all double quotes in audit messages (and maybe others for consistency) to single-quotes so that they don't need to be escaped in json serialization. Just a small QoL change. WDYT?

Reply:
eh... I don't care much one way or the other